easybuild/easyconfigs/p/PyMOL/PyMOL-2.4.1_198-foss-2019b.eb

```diff
@@ -0,0 +1,36 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Tarball'
+
+name = 'PyMOL'
+version = '2.4.1_198'
+subversion = '-Linux-x86_64-py37'
+pythonver = '3.7.4'
+
+homepage = 'https://pymol.org'
+description = """PyMOL is a user-sponsored molecular visualization system on an open-source foundation,
+maintained and distributed by Schrödinger."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
Since it's a binary release with it's own python, perhaps just have it at SYSTEM level instead?

Reply:
done